.github/workflows/build_prs_jest_report.yaml

```diff
@@ -33,16 +33,23 @@ jobs:
           node-version: ${{ matrix.node-version }}
           cache: 'yarn'
       - name: Tests on ${{ matrix.node-version }}
+        id: coverage
         uses: ArtiomTr/jest-coverage-report-action@v2
         with:
           annotations: none
           package-manager: yarn
           github-token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
I don't have quite enough context here, but could we use the default github.token here as mentioned in the [documentation](https://github.com/ArtiomTr/jest-coverage-report-action#custom-token):

> By default, this action takes github.token variable to publish reports on your PR.

I couldn't quickly find any info on what this token is or what rights it has.

```suggestion
```

Reply:
So, This is the one token you don't have access to when you're an external PR. It is a token that has write rights to REPO, which is why you don't want external PRs to have it. So, I do believe that for the markdown-report publish it should work without the token, but I'm not 100%

Reply:
Great! In that case, let's try it without. Doesn't work now regardless, so might as well give it a go? @gastonfournier ?

Reply:
the secret.GITHUB_TOKEN and github.token are equivalent from what I have read. I think we need to skip the usage of this in prs if we can, at least from external forks.